demo/predict-taxi-trip-duration-nb/Dockerfile

```diff
@@ -3,13 +3,14 @@ FROM openjdk:11.0.13-jre-slim-bullseye
 LABEL org.opencontainers.image.source https://github.com/4paradigm/OpenMLDB
 
 RUN  apt-get update \
-     && apt-get install -y --no-install-recommends libgomp1 curl binutils procps python3 python3-pip python3-numpy \
+     && apt-get install -y --no-install-recommends libgomp1 curl binutils procps python3 python3-pip python3-numpy vim \
```

Comment:
no vim !

Reply:
@tobegit3hub advice adding vim for user editing conf

Reply:
if a vimer, just install manually.  there is nano or vi already

Reply:
well, if vim is used in our quick start, then ..